web/src/containers/IncentivePayments.js

```diff
@@ -36,18 +34,31 @@ class IncentivePayments extends Component {
       <Fragment>
         {years.map(year => (
           <Fragment key={year}>
-            <h3 id={thId(year)}>{t('ffy', { year })}</h3>
-            <table className="table-cms table-fixed">
+            <table className="budget-table">
+              <caption className="ds-u-visibility--screen-reader">
+                {t('ffy', { year })}
+              </caption>
               <thead>
                 <tr>
-                  <th />
+                  <th>
+                    <span aria-hidden="true">
+                      {t('ffy', { year })}
+                    </span>
```

Comment:
In lines 38-46, we're creating a table caption that will be read by screen readers once, prior to reading the table contents, rather than creating a situation where the screen reader will say `FFY 2019` before _every single cell_.